test/run_pass/test_pwm.js

```diff
@@ -19,12 +19,17 @@ var Pwm = require('pwm');
 
 var pwm = new Pwm();
 
-var configuration = {};
+var configuration = {
+  period: 0.001,  // 1kHz
+  dutyCycle: 0.2  // 20%
+};
 
 if (process.platform === 'linux') {
   configuration.pin = 0;
 } else if (process.platform === 'nuttx') {
   configuration.pin = require('stm32f4dis').pin.PWM1.CH1_1;
+} else if (process.platform === 'tizenrt') {
+  configuration.pin = require('artik053').pin.PWM0;
```

Comment:
I think it is useful for end developer. It is looks better when you have: `open({pin: pin.PWM0})` than `open({pin: 0})`. This is meaningful, easy to read ID instead of hardcoded magic number. Additionally it is consistent with stm.